src/Raven.Server/Documents/ConflictsStorage.cs

```diff
@@ -841,7 +841,7 @@ public string GetCollection(DocumentsOperationContext context, string id)
         [StorageIndexEntryKeyGenerator]
         internal static ByteStringContext.Scope GenerateBucketAndEtagIndexKeyForConflicts(ByteStringContext context, ref TableValueReader tvr, out Slice slice)
         {
-            return GenerateBucketAndEtagIndexKey(context, idIndex: (int)ConflictsTable.LowerId, etagIndex: (int)ConflictsTable.Etag, ref tvr, out slice);
+            return ShardedDocumentsStorage.GenerateBucketAndEtagIndexKey(context, idIndex: (int)ConflictsTable.LowerId, etagIndex: (int)ConflictsTable.Etag, ref tvr, out slice);
```

Comment:
let's move this and all sharded related functions into a new partial class `ConflictsStorage.Sharding`

Reply:
done